test/fixtures/sub-process.js

```diff
@@ -4,7 +4,15 @@ var Insight = require('../../lib');
 var insight = new Insight({
 	packageName: 'yeoman',
 	packageVersion: '0.0.0',
-	trackingCode: 'GA-1234567-1'
+	trackingCode: 'GA-1234567-1',
+	config: {
```

Comment:
Not really, thing skipped in review would probably never end up being done. This should be fairly easy for you to test here.